src/factories/workflow-specific/Immutable_PIM_Factory_v1.sol

```diff
@@ -0,0 +1,184 @@
+import {IBondingCurveBase_v1} from
+    "@fm/bondingCurve/interfaces/IBondingCurveBase_v1.sol";
+import {IModule_v1} from "src/modules/base/IModule_v1.sol";
+
+// Internal Implementations
+import {FM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1} from
+    "@fm/bondingCurve/FM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1.sol";
+
+// External Interfaces
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+
+// External Implementations
+import {ERC20Issuance_v1} from "src/external/token/ERC20Issuance_v1.sol";
+
+// External Dependencies
+import {ERC2771Context, Context} from "@oz/metatx/ERC2771Context.sol";
+
+contract Immutable_PIM_Factory_v1 is
+    ERC2771Context,
+    IImmutable_PIM_Factory_v1
+{
+    //--------------------------------------------------------------------------
+    // State Variables
+
+    /// @dev	store address of {Orchestratorfactory_v1}.
+    address public orchestratorFactory;
+
+    /// @dev	mapping of Funding Manager address to `feeRecipient` address.
+    mapping(address fundingManager => address feeRecipient) private
+        _pimFeeRecipients;
+
+    //--------------------------------------------------------------------------
+    // Modifiers
+
+    /// @dev	Modifier to guarantee the caller is the fee recipient for the given funding manager.
+    modifier onlyPimFeeRecipient(address fundingManager) {
+        if (_msgSender() != _pimFeeRecipients[fundingManager]) {
+            revert PIM_WorkflowFactory__OnlyPimFeeRecipient();
+        }
+        _;
+    }
+
+    //--------------------------------------------------------------------------
+    // Constructor
+
+    constructor(address _orchestratorFactory, address _trustedForwarder)
+        ERC2771Context(_trustedForwarder)
+    {
+        orchestratorFactory = _orchestratorFactory;
+    }
+
+    //--------------------------------------------------------------------------
+    // Public Mutating Functions
+
+    /// @inheritdoc IImmutable_PIM_Factory_v1
+    function createPIMWorkflow(
+        IOrchestratorFactory_v1.WorkflowConfig memory workflowConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory fundingManagerConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory authorizerConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory paymentProcessorConfig,
+        IOrchestratorFactory_v1.ModuleConfig[] memory moduleConfigs,
+        IBondingCurveBase_v1.IssuanceToken memory issuanceTokenParams,
+        uint initialPurchaseAmount
+    ) external returns (IOrchestrator_v1 orchestrator) {
+        // deploy issuance token
+        ERC20Issuance_v1 issuanceToken = new ERC20Issuance_v1(
+            issuanceTokenParams.name,
+            issuanceTokenParams.symbol,
+            issuanceTokenParams.decimals,
+            issuanceTokenParams.maxSupply,
+            address(this) // assigns owner role to itself initially to manage minting rights temporarily
+        );
+        // MODIFY AUTHORIZER CONFIG
```

Comment:
A line break between logical sections would make sense IMHO, so here before this like (`// MODIFY AUTHORIZER CONFIG`), then again before `// MODIFY AUTHORIZER CONFIG`, etc.